forbidigo/forbidigo.go

```diff
@@ -57,7 +65,7 @@ type Linter struct {
 }
 
 func DefaultPatterns() []string {
-	return []string{`^(fmt\.Print(|f|ln)|print|println)$`}
+	return []string{`^(?P<pkg>(fmt)\.Print(|f|ln)|print|println)$`}
```

Comment:
Shouldn't this be `^((?P<pkg>fmt)\.Print(|f|ln)|print|println)$`?

Reply:
(I think we also repeat this pattern in the README).

Reply:
Yes, that was the intention. Will fix.

The reason why it works is that the name subgroup doesn't have to be the one which matches the package. Nothing cares about that. So this would also work:
```
^(?P<pkg>fmt\.Print(|f|ln)|print|println)$
```

I was wondering whether I should explicitly document this, but then decided against it because the intent of a pattern is more obvious when doing it as you said.